workflows/execute_workflows.sh

```diff
@@ -5,7 +5,42 @@ WORKFLOW_DIR="$ROOT"/workflows
 OCRD_WORKFLOW_DIR="$WORKFLOW_DIR"/ocrd_workflows
 WORKSPACE_DIR="$WORKFLOW_DIR"/workspaces
 
-mkdir $WORKSPACE_DIR
+adjust_workflow_settings() {
+    # $1: $WORKFLOW
+    # $2: $DIR_NAME
+    sed -i "s INPUT_DIR . g" "$1"
```

Comment:
Using space as the delimiter in sed is wild. I tend to use comma or slash but of course ` ` is a valid choice, just never occured to me :)